Lib/ntpath.py

```diff
@@ -110,34 +110,34 @@ def join(path, *paths):
     try:
         if not paths:
             path[:0] + sep  #23780: Ensure compatible data type even if p is null.
-        result_drive, result_root, result_path = splitroot(path)
+        result_drive, result_root, result_tail = splitroot(path)
```

Comment:
This is also a stylistic change, having no effect on the actual operation of the function. These sorts of changes aren't usually accepted.